io/config.go

```diff
@@ -20,9 +20,10 @@ package io
 
 import (
 	"encoding/json"
-	"github.com/apache/brooklyn-client/error_handler"
 	"os"
 	"path/filepath"
+
+	"github.com/apache/brooklyn-client/error_handler"
```

Comment:
This came from the `gofmt` tool used by the vim-go plugin. It organizes imports automatically by path and third party packages are separated with blank line from standard library packages.